samples/gpu-passthrough.yaml

```diff
@@ -0,0 +1,29 @@
+# ###
+# This example shows how to use the orka-full Task to create a single macOS VM
+# that uses the host's GPU. GPU Passthrough needs to be enabled in the Orka cluster.
+# The script provided in the params will be executed inside the VM.
+#
+# You will need to have a secret containing the Orka credentials as well as a
+# secret containing the VM SSH credentials as described in the README.
+# ###
+---
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: gpu-passthrough
+spec:
+  taskRef:
+    name: orka-full
+  params:
+    - name: base-image
+      value: catalina-ssh-30G.img
```

Comment:
It's probably worth using a "stock" image here ... for example, `90GMontereySSH.img`. Same thing with the other samples